source/_integrations/alexa.intent.markdown

```diff
@@ -94,8 +94,9 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
 - Find the skill you just created and click `Edit` in the `Actions` column.
 - Click `ACCOUNT LINKING` in the left navigation bar of build page
 - Input all information required. Assuming your Home Assistant can be accessed by https://[YOUR HOME ASSISTANT URL:PORT]
-  * `Authorization URI`: https://[YOUR HOME ASSISTANT URL:PORT]/auth/authorize
-  * `Access Token URI`: https://[YOUR HOME ASSISTANT URL:PORT]/auth/token
+  * `Authorization URI`: https://[YOUR HOME ASSISTANT URL]/auth/authorize
```

Comment:
How does that work if the port is no longer mentioned?

Reply:
It's not needed because it has to be the standard HTTPS port (443).